StripeFinancialConnections/StripeFinancialConnections/Source/Native/LinkAccountPicker/LinkAccountPickerViewController.swift

```diff
@@ -45,7 +45,7 @@ final class LinkAccountPickerViewController: UIViewController {
     private weak var bodyView: LinkAccountPickerBodyView?
     private lazy var footerView: LinkAccountPickerFooterView = {
         return LinkAccountPickerFooterView(
-            isStripeDirect: dataSource.manifest.isStripeDirect ?? false,
+            isStripeDirect: false,
```

Comment:
Stripe.js does this (likely because we don't want to show Stripe in one place, and Link in another):
- https://git.corp.stripe.com/stripe-internal/stripe-js-v3/blob/957a6a341820b5872640c38d3caec7022d7a04c3/src/linkedAccounts/inner/components/panes/LinkAccountPicker/index.tsx#L51